cypress-tests/cypress/support/pageBuilder/pbCreateCategoryAndBlocks/createHeadingBlock.ts

```diff
@@ -0,0 +1,29 @@
+export const createHeadingBlock = (blockText: string) => {
+    return {
```

Comment:
Let's include the 

```ts
{
                type: "block",
                data: {},
                elements: [createHeadingBlock(contentText)]
            };
```

part here, instead of having it in `pbCreateCategoryAndBlocks.ts`. It's just that the block is incomplete without the wrapper. Having all in place might be easier to grasp, and also reuse if needed in the future.